agent_application/docker/compose.yaml

```diff
@@ -26,7 +26,7 @@ services:
     depends_on:
       - cqrs-postgres-db
   ssi-agent:
-    image: impiercetechnologies/ssi-agent:837870c
+    image: impiercetechnologies/ssi-agent:1.0.0-beta.9
```

Comment:
I think we should generally not bake a version that does not exist yet into any file. I know it makes sense to have it "all in one PR", but it breaks with semantic release principles. A meta tag such as `latest` would be a lot cleaner (although we don't have it yet). Let's rather add a comment in the compose file and force the integrator to make a choice themselves.